visallo/visallo
# Added: Extensible general settings page and timezone configuration

This change adds a general settings page to the user account modal dialog.
This change also adds a front end setting to change the timezone for all
front end times.

![image](https://cloud.githubusercontent.com/assets/808857/26246570/d4d20a62-3c67-11e7-8d86-06ab1edc100a.png)

Testing Instructions:
- Have entities with time properties. Open the user account page and change the timezone. Verify that the times have changed correctly to the new timezone.

CHANGELOG
Added: Extensible general settings page to the account modal
Added: Front end timezone configuration
